ipauser: Add support to modify GECOS field.

rjeffman opened the pull request:
This patch adds a new parameter to ipauser, 'gecos', which can be used to set the 'gecos' field of an IPA user. The default behavior of automatically set the GECOS field to "<first> <last>" is not modified, it is only possible to change the field to a custom value.

No validation on the value provided is done, as it is with FreeIPA.

varunmylaraiah reviewed (approved):
Thanks for the PR.

I have tested it, and it's working fine.

t-woerner commented:
Please add the information about gecos to the README also.

t-woerner commented:
Is it possible to reset gecos with an empty string?

rjeffman commented:
As GECOS is just another user information field, I added the field to an existing user creation example (pinky).

Yes, it is possible to clear GECOS with an empty string. I added tests for that scenario.
